app/src/main/java/gateway/controller/CtrlUpdatePassword.java

```diff
@@ -0,0 +1,86 @@
+package gateway.controller;
+
+import gateway.config.Config;
+import gateway.services.ServiceAuth;
+import gateway.services.UtilValidator;
+import gateway.soap.request.ReqAccPassword;
+import gateway.soap.response.ResFileDownload;
+import gateway.soap.response.ResSession;
+import gateway.soap.response.ResStatus;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import org.json.JSONObject;
+
+public class CtrlUpdatePassword
+{
+	public static ResStatus account_password (ReqAccPassword accPassword)
+	{
+
+		// Create a new ResSession object to hold the response data
+		ResSession resSession = new ResSession ();
+
+		// Check fields of ReqAccPassword
+		ResStatus resValidate = UtilValidator.validate (accPassword);
+		if (resValidate.error) {
+			return ResStatus.downCast (ResFileDownload.class, resValidate);
```

Comment:
Since account_password returns a ResStatus you can return the ResStatus received unchanged. No need to downcast it.